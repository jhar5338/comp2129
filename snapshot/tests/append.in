append a 1
get a
set a
get a
append a
get a
set a 1
append a 2 3 4 5
get a
append b
bye
