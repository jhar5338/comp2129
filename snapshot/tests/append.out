> no such key

> no such key

> ok

> []

> ok

> []

> ok

> ok

> [1 2 3 4 5]

> no such key

> bye
